Grid-AdaptiveLayout/Grid-AdaptiveLayout/Shared/NavMenu.razor

```diff
@@ -16,7 +16,7 @@
         </div>
         <div class="nav-item px-3">
             <NavLink class="nav-link" href="counter">
-                <span class="oi oi-plus" aria-hidden="true"></span> Counter
+                <span class="oi oi-plus" aria-hidden="true"></span> AdaptiveUI with SfMediaQuery
```

Comment:
Enable/Disable DataGrid Adaptive UI using Media Query Component

Reply:
Changed the side bar name as DataGrid Adaptive UI using Media Query
https://github.com/SyncfusionExamples/blazor-datagrid-render-adaptive-layout/commit/9b4416874df66fe6c2afe45f1b0933f7d4b188f7